### PR TITLE
Set up CI to lint workflows with action-validator

### PR DESCRIPTION
**Pull request by Johennes**
The idea for this came out of a chat with @t3chguy (and me continuously nuking `develop`)

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
         "lint:types:src": "tsc --noEmit --jsx react",
         "lint:types:module_system": "tsc --noEmit --project ./tsconfig.module_system.json",
         "lint:style": "stylelint \"res/css/**/*.pcss\"",
+        "lint:workflows": "find .github/workflows -type f \\( -iname '*.yaml' -o -iname '*.yml' \\) | xargs -I {} sh -c 'echo \"Linting {}\"; action-validator \"{}\"'",
```
Comment (t3chguy) [inline package.json:63]: Why not something like `action-validator .github/workflows/*.{yaml,yml}`

**Review by t3chguy — COMMENTED** [review]
What about our other repos?

File: package.json (hunk 1 above)
Comment (Johennes) [reply package.json:63]: That would be much easier, yeah. But, unfortunately, it only supports a single file argument. 😕

**Comment by Johennes** [comment]
> What about our other repos?

Yes, they should receive the same check. element-meta should, too. Just wanted to get agreement on how to set it up before then copying it over.

File: package.json (hunk 1 above)
Comment (t3chguy) [reply package.json:63]: The help and testing implies otherwise

image

Comment (t3chguy) [reply package.json:63]: https://github.com/mpalmer/action-validator/blob/main/src/config.rs#L18 it takes a vector of paths

Comment (Johennes) [reply package.json:63]: I think that might not have been released? The version I installed through yarn bails when specifying more than one file.

Comment (t3chguy) [reply package.json:63]: Ah maybe its not on yarn, when installing via cargo I can do multiple

Comment (Johennes) [reply package.json:63]: Want me to run it through cargo from within the yarn script? The xargs thing doesn't strike me as overly bad but I don't feel strongly either way.

Comment (t3chguy) [reply package.json:63]: don't feel strongly either way

Comment (Johennes) [reply package.json:63]: As discussed in DM, given that we don't appear to be using cargo for anything else outside the CI yet and that adding both the install and run into the script might not be much nicer than the xargs thing, we'll keep the xargs variant for now. Adding multi-file support is tracked upstream in https://github.com/mpalmer/action-validator/issues/41. Once this is fixed, we can then easily simplify the yarn script later.

Comment (Johennes) [reply package.json:63]: @t3chguy given the above, I think this is good to land?

**Review by t3chguy — APPROVED** [review]

**Pull request merged**